docs/content/guide/migration.ngdoc

```diff
@@ -13,6 +13,156 @@ which drives many of these changes.
 * Several new features, especially animations, would not be possible without a few changes.
 * Finally, some outstanding bugs were best fixed by changing an existing API.
 
+
+
+
+
+# Migrate from 1.4 to 1.5
+
+Angular 1.5 takes a big step towards enabling developers to prepare themselves for a smoother
+transition to Angular 2 in the future. Architecturing your applications using components, making use
+of lifecycle hooks in directive controllers and relying on native ES6 features (such as classes and
+arrow functions) are now all possible with Angular 1.5.
+
+
+This release includes numerous bug and security fixes as well as performance improvements to core
```

Comment:
I added the comma (but not the `also` - it sounded better without it to me :smiley:).